src/api/status/public/js/build-log/datetime-calculator.js

```diff
@@ -0,0 +1,39 @@
+const calculatorData = document.getElementById('calculatorData');
+const buildIcon = document.getElementById('build-icon');
+
+function getBuildDuration(startedAtValue, stoppedAtValue) {
+  let diff;
+  if (stoppedAtValue === null) {
+    const date = new Date(startedAtValue);
+    const currentDate = new Date();
+    diff = currentDate - date;
+  } else {
+    const date = new Date(startedAtValue);
+    const currentDate = new Date(stoppedAtValue);
+    diff = currentDate - date;
+  }
+
+  diff = (diff / 1000).toFixed();
+  const hours = Math.floor(diff / 3600) % 24;
+  const minutes = Math.floor(diff / 60) % 60;
+  const seconds = diff % 60;
+  return [hours, minutes, seconds];
+}
+
+export default function renderBuildInfo(data) {
+  const [startedAtValue, stoppedAt, result] = data;
+  const [hours, minutes, seconds] = getBuildDuration(startedAtValue, stoppedAt);
+
+  if (startedAtValue !== null && stoppedAt === null) {
+    buildIcon.innerHTML = 'motion_photos_on';
+    buildIcon.classList.add('fa-spin', 'text-warning', 'fa-12x');
+  } else if (result === 0 && stoppedAt !== null) {
+    buildIcon.innerHTML = 'check_circle';
+    buildIcon.classList.add('text-success');
+  } else if (result !== 0 && stoppedAt !== 'null') {
+    buildIcon.innerHTML = 'cancel';
+    buildIcon.classList.add('text-danger');
+  }
```

Comment:
These are the needed changes. Move these inside the function that creates the build-header. 

Screenshots or videos for documenting new UI changes are much appreciated (inside PR description).

Reply:
you want me to also delete the `datetime-calculator.js` ?

Reply:
Yeah, we won't need it.